config/mail.php

```diff
@@ -4,45 +4,91 @@
+    |            "failover", "roundrobin"
     |
     */
 
-    'driver' => env('MAIL_DRIVER', 'log'),
-    'host' => env('SMTP_HOST', 'localhost'),
-    'port' => env('SMTP_PORT', 25),
-    'encryption' => env('SMTP_ENCRYPTION', false),
-    'username' => env('SMTP_USERNAME'),
-    'password' => env('SMTP_PASSWORD'),
+    'mailers' => [
+
+        'smtp' => [
+            'transport' => 'smtp',
+            'url' => env('MAIL_URL'),
+            'host' => env('SMTP_HOST', '127.0.0.1'),
+            'port' => env('SMTP_PORT', 2525),
+            'encryption' => env('SMTP_ENCRYPTION', false),
+            'username' => env('SMTP_USERNAME'),
+            'password' => env('SMTP_PASSWORD'),
+            'timeout' => null,
+            'local_domain' => env('MAIL_EHLO_DOMAIN', parse_url(env('APP_URL', 'http://localhost'), PHP_URL_HOST)),
+        ],
+
+        'log' => [
+            'transport' => 'log',
+            'channel' => env('MAIL_LOG_CHANNEL'),
+        ],
+
+        'array' => [
+            'transport' => 'array',
+        ],
+    ],
 
     /*
     |--------------------------------------------------------------------------
     | Global "From" Address
     |--------------------------------------------------------------------------
     |
-    | You may wish for all e-mails sent by your application to be sent from
-    | the same address. Here, you may specify a name and address that is
-    | used globally for all e-mails that are sent by your application.
+    | You may wish for all emails sent by your application to be sent from
+    | the same address. Here you may specify a name and address that is
+    | used globally for all emails that are sent by your application.
     |
     */
 
-    'from' => ['address' => 'haveyoutriedturningitoffandonagain@proto.utwente.nl', 'name' => 'S.A. Proto Have You Tried Turning It Off And On Again committee'],
+    'from' => [
+        'address' => env('MAIL_FROM_ADDRESS', 'haveyoutriedturningitoffandonagain@proto.utwente.nl'),
+        'name' => env('MAIL_FROM_NAME', 'S.A. Proto Have You Tried Turning It Off And On Again committee'),
+    ],
 
     /*
-     * In order to provide support for Laravel 5.4's new Markdown mail components,
-     * you should add the following block of configuration to the bottom of your
-     * mail configuration file:
-     */
+    |--------------------------------------------------------------------------
+    | Markdown Mail Settings
+    |--------------------------------------------------------------------------
+    |
+    | If you are using Markdown based email rendering, you may configure your
+    | theme and component paths here, allowing you to customize the design
+    | of the emails. Or, you may simply stick with the Laravel defaults!
+    |
+    */
 
     'markdown' => [
-        'theme' => 'default',
+        'theme' => env('MAIL_MARKDOWN_THEME', 'default'),
+
         'paths' => [
             resource_path('views/vendor/mail'),
         ],
```

Comment:
Where does this come from?
I dont see it in the laravel 11 mail config?

https://github.com/laravel/laravel/blob/11.x/config/mail.php

Reply:
This came from the config in our laravel vendor folder